dynamic_rest/serializers.py

```diff
@@ -66,19 +66,16 @@ def id_only(self):
     @property
     def data(self):
         """Get the data, after performing post-processing if necessary."""
-        if not hasattr(self, '_sideloaded_data'):
+        if not hasattr(self, '_processed_data'):
             data = super(DynamicListSerializer, self).data
-            if self.child.sideload:
-                self._sideloaded_data = ReturnDict(
-                    SideloadingProcessor(
-                        self,
-                        data
-                    ).data,
-                    serializer=self
-                )
-            else:
-                self._sideloaded_data = ReturnList(data, serializer=self)
-        return self._sideloaded_data
+            self._processed_data = ReturnDict(
```

Comment:
Technically a breaking change, since this means that we will no longer return "data" without an envelope if a viewset requests that it should be returned without an envelope, if a viewset had sideloading disabled entirely.